src/bind.rs

```diff
@@ -145,24 +151,6 @@ pub struct RebindTls<B> {
     endpoint: Endpoint,
 }
 
-pub type Service<B> = BoundService<B>;
-
-pub type Stack<B> = WatchService<tls::ConditionalClientConfig, RebindTls<B>>;
-
-type ReconnectStack<B> = Reconnect<NewHttp<B>>;
-
-pub type NewHttp<B> = orig_proto::Upgrade<NormalizeUri<telemetry::http::service::NewHttp<Client<B>, B, HttpBody>>>;
-
-pub type HttpResponse = http::Response<telemetry::http::service::ResponseBody<HttpBody>>;
```

Comment:
Getting rid of the `HttpRequest`/`HttpResponse` aliases is great --- my experience has been that they make writing code in this module (and code that interacts with this module) more confusing.

In particular, it wasn't immediately clear that the meaning of the aliases was that the response had been wrapped with telemetry sensors, so I suspect newcomers wouldn't have immediately understood when to take a `http::Request` and when to take a `bind::HttpRequest`...